src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/HenleggBehandling/HenleggBehandling.tsx

```diff
@@ -45,10 +44,8 @@ const Knapperad = styled.div`
     justify-content: end;
 `;
 
-const StyledLenke = styled(Lenke)<{ visLenke: boolean }>`
+const StyledLenke = styled(Link)`
```

Comment:
Endret logikken for `visLenke` til conditional render av hele lenken. Flex er ikke nødvendig når den ikke har flere barneelementer